ports/stm32/usbd_conf.c

```diff
@@ -75,6 +75,16 @@ void HAL_PCD_MspInit(PCD_HandleTypeDef *hpcd) {
         mp_hal_pin_config(pin_A12, MP_HAL_PIN_MODE_ANALOG, MP_HAL_PIN_PULL_NONE, 0);
         mp_hal_pin_config_speed(pin_A12, GPIO_SPEED_FREQ_VERY_HIGH);
 
+        #elif defined(STM32L1)
+
+        // STM32L1 doesn't have an alternate function for USB.
+        // To be disconnected from all peripherals, put in input mode.
+
+        mp_hal_pin_config(pin_A11, MP_HAL_PIN_MODE_INPUT, MP_HAL_PIN_PULL_NONE, 0);
```

Comment:
Is it possible to use `MP_HAL_PIN_MODE_ANALOG` here, instead of input mode?  That's how the G0 and G4 MCU's work.

Reply:
I tried it before but it does not work with STM32L1. 
This is due to difference of the the port mode's reset state: G0 and G4 MCUs are analog mode, while L1 MCUs are input mode.

Reply:
OK, thanks for trying that.